### PR TITLE
Update.typography

### PR DESCRIPTION
**Pull request by herman-rogers**
- Adding updated typography
- Add new quarternary button
- Aligning other stories with new typography

### CODE REVIEW

File: src/theme/ThemeTypography.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```
Comment (lanwen) [inline src/theme/ThemeTypography.jsx]: is it autogenerated?

File: src/button/button.style.js [hunk 2]
```diff
@@ -109,6 +109,42 @@ const ButtonStyles = styled.div`
...
+  .quarternary {
+    ${baseButton} width: 178px;
+    height: 40px;
+    background-color: #fff;
+    border: solid 2px #898988;
+    color: #898988;
+    font-size: 15px;
+    font-weight: 500;
+    cursor: pointer;
+    transition: color 0.2s ease, background-color 0.2s ease;
```
Comment (lanwen) [inline src/button/button.style.js]: is it better to move it to the base?

File: src/theme/ThemeTypography.jsx (hunk 1 above)
Comment (herman-rogers) [reply src/theme/ThemeTypography.jsx]: This file isn't auto generated

Comment (lanwen) [reply src/theme/ThemeTypography.jsx]: then maybe better to use variables for colors, heights? Or the numbers is completely random and can't be counted from some base value?

File: README.md [hunk 3]
```diff
@@ -16,6 +16,14 @@ or
 
 To serve the storybook just run `npm run storybook` which will serve locally on your machine via localhost.
 
+## Design
+
+For the most part all designs laid out in this component library are spearheaded by the Vivy design team - therefore we take
+a lot of our cues directly from the design team. In case a design does not exist grab a design and talk over the implementation
+details with them. You can always create a quick placeholder - but official design must be signed off and accepted by the design team.
+
+You can find all current, updated, and new designs in the [Vivy zeplin web project](https://app.zeplin.io/project/5a8d80ddc1a5d9014b9a9b29)
```
Comment (lanwen) [inline README.md]: Is that link authenticated?

File: README.md [hunk 4]
```diff
@@ -16,6 +16,14 @@ or
 
 To serve the storybook just run `npm run storybook` which will serve locally on your machine via localhost.
 
+## Design
+
+For the most part all designs laid out in this component library are spearheaded by Vivy design - therefore we take
+a lot of our design cues directly from them. In case a design does not exist, grab a designer and talk over the implementation
+details with them. You can always create a quick placeholder - but official design must be signed off and accepted by the design team.
+
+You can find all current, updated, and new designs in the [Vivy zeplin web project](https://app.zeplin.io/project/5a8d80ddc1a5d9014b9a9b29)
```
Comment (lanwen) [inline README.md]: Is that link authenticated?

**Review by lanwen — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (herman-rogers) [reply README.md]: I believe you need to sign in with zeplin in order to see it, should I redact the project/id from the url?

Comment (lanwen) [reply README.md]: Yea, probably better to be not so transparent

Comment (herman-rogers) [reply README.md]: Cool

**Pull request merged**